### DISCUSSION TITLE
bug: shows wrong rank

### DISCUSSION THREAD

**Discussion by hashfx** [anuraghazra/github-readme-stats#3386] [Q&A] [answered]
### Describe the bug

Shows that I'm in Top 40.2% of GitHub users with 336 commits
Also shows that I'm in Top 42.2 with 977 commits [private commits counted]

With more number of commits, I should have a rank of less number (<40.2%)

### Expected behavior

977 > 336
But in Ranking System (40>42 {because it's a rank})

So with 977 commits, I should be given a ranking more than that of 336 commits
That means I should be in less than 40.2% of total GitHub users

### Screenshots / Live demo link

![bug](https://github.com/anuraghazra/github-readme-stats/assets/69109482/0dfc1e68-92b9-40c6-9868-c5c4e63aa064)

**Comment by qwerty541** [comment, marked as answer]
Hey, @hashfx! Thanks for your question! During rank calculation cmmits count have four time lower value if you're using query parameter `&include_all_commits=true`. Therefore having 250 commits per year and 1000 commits lifetime is equal for rank calculation. Since you have more commits per year, it shows more higher rank. You can check our rank calculation algorithm here https://github.com/anuraghazra/github-readme-stats/blob/master/src/calculateRank.js

**Reply by hashfx** [reply]
Cool, understandable 👍🏻
